source/scpd/scp/SCPDriver.d

```diff
@@ -71,18 +71,37 @@ nothrow:
     // returning Value() means no valid value could be extracted
     Value extractValidValue(uint64_t slotIndex, ref const(Value) value);
 
-    // `getValueString` is used for debugging
-    // default implementation is the hash of the value
-    //std::string getValueString(ref const(Value) v) const;
-    void* getValueString(ref const(Value) v) const;
-
-    // `toStrKey` returns StrKey encoded string representation
-    //std::string toStrKey(ref const(PublicKey) pk, bool fullKey = true) const;
-    void* toStrKey(ref const(PublicKey) pk, bool fullKey = true) const;
-
-    // `toShortString` converts to the common name of a key if found
-    //std::string toShortString(ref const(PublicKey) pk) const;
-    void* toShortString(ref const(PublicKey) pk) const;
+    version(Windows)
+    {
+        // TODO: Take temporary action to support mangling in Windows MSVC
```

Comment:
Can you add a comment explaining *why* we need to take those actions on Windows and not POSIX ?